ouroboros-network/test/Test/Ouroboros/Network/TipSample/Client.hs

```diff
@@ -0,0 +1,608 @@
+{-# LANGUAGE BangPatterns        #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE DerivingVia         #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE KindSignatures      #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TupleSections       #-}
+
+module Test.Ouroboros.Network.TipSample.Client (tests) where
```

Comment:
Is this file intended to be part of this PR? I don't see it being added to the cabal file nor added to the lists of tests to run.

Reply:
thanks for catching this, no it should not.